packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -554,4 +540,24 @@ -(void)copyToClipboard:(NSString *)content {
   clipboard.string = (content ? : @"");
 }
 
+- (void)setDevMenuAppBridge
+{
+  DevMenuManager *manager = [DevMenuManager shared];
+  manager.currentBridge = self.appBridge;
+  
+  if (self.manifest != nil) {
+    // TODO - update to proper values / convert via instance method
+    manager.currentManifest = [self.manifest.rawManifestJSON copy];
+    manager.currentManifestURL = self.manifestURL;
+  }
+}
+
+- (void)invalidateDevMenuApp
+{
+  DevMenuManager *manager = [DevMenuManager shared];
+  manager.currentBridge = nil;
+  manager.currentManifest = @{};
```

Comment:
do you think it would be reasonable to make this field nullable and have `nil` indicate no manifest?

Reply:
yes that seems reasonable!